articles/what-is-a-custom-domain.mdx

```diff
@@ -0,0 +1,64 @@
+---
+_schema: default
+_uuid: 5804dd03-210e-48cc-a02f-1b29f439ce0e
+_created_at: 2024-11-22T14:15:34+13:00
+title: What is a Custom Domain?
+nav_title: What is a Custom Domain?
+published: true
+image: >-
+  https://cc-dam.imgix.net/documentation/images/default-thumbnail/2023-Q4/CloudCannon-Documentation-Thumbnail.png
+description: Learn about Custom Domains, subdomains, and subpaths.
+article_category:
+  - Explanation
+article_topic:
+  - Custom Domains
+tags: []
+related_articles: []
+related_links: []
+explicit_canonical:
+---
+A Custom Domain is a unique address used to identify a website. For example, CloudCannon's domain is [cloudcannon.com](https://cloudcannon.com/). The domain is the part of your URL that users are most likely to remember. You can purchase a Domain Name from a Domain Registrar.
+
+* **Domain Name** — The unique address used to access a website (e.g., [cloudcannon.com](https://cloudcannon.com/)).
+* **Domain Registrar** — A third-party company that handles the reservation of domain names and assigns IP addresses to domain names. Most CloudCannon users rent a domain name from a Registrar (e.g., <a href="https://www.cloudflare.com/en-gb/products/registrar/" target="_blank" rel="noopener">Cloudflare Registrar</a>, <a href="https://www.namecheap.com/" target="_blank" rel="noopener">NameCheap</a>).
```

Comment:
`assigns IP addresses to domain names` is the responsibility of a DNS provider.
The Domain Registrar will assign a Nameserver, which is the DNS provider. 
Domain Registrar could ALSO be used as the DNS provider, but they are separate offerings.